run_homepage_tests.sh

```diff
@@ -0,0 +1 @@
+site/vendor/bin/phpunit -c site/tests/phpunit.xml --filter testHomePageView
```

Comment:
Why are we only filtering on this test? I don't think this should be merged into the submitty master branch.

Reply:
No this is only to run homepage tests